docs/make.jl

```diff
@@ -1,5 +1,14 @@
 using Documenter, FiniteDifferences
 
+# Copy the README.
+repo_root = dirname(dirname(@__FILE__))
+cp(
+    joinpath(repo_root, "README.md"),
+    joinpath(repo_root, "docs", "src", "index.md");
+    # `index.md` will already exist, so we set `force=true` to overwrite it.
```

Comment:
we should delete and add it to `.gitignore` it, or someone will re-add it by mistake,.